04_data-viz.qmd

```diff
@@ -391,40 +436,56 @@ tibble(x = 1:100,
 
 ```
 
-### `r kableExtra::text_spec("Exercise 5", color = "#1696d2")`
 
-**`r kableExtra::text_spec("Step 1:", color = "#55b748")`** Create a scatter plot of the `storms` data set with `pressure` on the x-axis and `wind` on the y-axis. 
+::: callout
 
-**`r kableExtra::text_spec("Step 2:", color = "#55b748")`** Add `facet_wrap(~ category)`
+#### [`r paste("Exercise", exercise_number)`]{style="color:#1696d2;"}
 
-\vspace{0.25in}
+```{r}
+#| echo: false
+
+exercise_number <- exercise_number + 1
+```
+
+1. Create a scatter plot of the `storms` data set with `pressure` on the x-axis and `wind` on the y-axis. 
+
+2. Add `facet_wrap(~ category)`
+
+:::
+
+### Facets:
 
 :::{.callout-note}
-**6 Facets (optional)** break data into meaningful subsets. `facet_wrap()`, `facet_grid()`, and `facet_geo()`.
+**Facets (optional)** break data into meaningful subsets. `facet_wrap()`, `facet_grid()`, and `facet_geo()`.
```

Comment:
Let's drop references to `facet_geo()` or move it to a footnote.

Reply:
Moved to footnote